static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -92,20 +91,15 @@ function ConsoleMessage({breadcrumb, isLast, startTimestamp = 0}: ConsoleMessage
         <MessageFormatter breadcrumb={breadcrumb} />
       </Message>
       <ConsoleTimestamp isLast={isLast} level={breadcrumb.level}>
-        <Hovercard
-          body={`${t('Relative Time')}: ${showPlayerTime(
-            breadcrumb.timestamp || '',
-            startTimestamp
-          )}`}
-        >
-          <DateTime
-            timeOnly
-            date={breadcrumb.timestamp}
+        <Tooltip title={<DateTime date={breadcrumb.timestamp} format="LTS" />}>
```

Comment:
Let's use a format that already exists in `<DateTime>` -- also we should probably display the date here. I think in the console it's a bit noisy, but in a tooltip users will want the extra details.

I think passing `seconds` prop is all we need.

Reply:
Changed and pushed, this is what it looks like now 
<img width="228" alt="image" src="https://user-images.githubusercontent.com/7014871/170083060-cae99590-090c-4f71-9bad-5268098a4080.png">

Reply:
lgtm